src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -62,4 +65,13 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.FSharp" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.FSharp.UnitTests" />
   </ItemGroup>
+
+  <!-- Turn off windows dependency check. Some packages still have windows-specific dependencies,
+       even though in the cases that matter they are not used at run-time -->
```

Comment:
Can you file a bug so we can track what these issues still are and link it here, or link to the existing issues?

Reply:
Yeah, I'll do some binlog analysis to see what is triggering it here.

Reply:
It was Microsoft.VisualStudio.Language.Intellisense. I've called it out in the csproj, but I don't think anything can be done about it for now.